docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -678,9 +689,136 @@ This could be done by assigning unique (or practically unique, with very low pro
+about immigration from ACS with information about emigration from demographic analysis
+(for those born outside the US) and analysis of foreign censuses (for those born in
+the US). [Census_PopEst_Methodology]_
+Without access to the source data, we cannot replicate these methods, which is why we
+use the published NIM values instead of directly estimating emigration.
+
+The NIM values are not published fully stratified.
+Out of the available stratifications, we chose to use the values stratified
+by (broad categories of) race/ethnicity, because these are most likely to have
+PRL implications.
+
+Inspired by the methodology of the PopEst team at the Census Bureau,
+we further stratify emigration by assuming that **emigrants** have the same
+characteristics as **immigrants**.
+There are clear reasons why this assumption would not be correct
+(e.g. the fact that the US is one of the wealthiest countries in the world means
+it is unlikely to have symmetric characteristics of incoming and outgoing migration)
+but it does likely capture some of the ways in which people with different characteristics
+have different propensities for international migration, regardless of origin/destination.
+
+First, we distribute emigration by move type, replicating the distribution of
+move type in each broad race/ethnicity group (non-Hispanic White alone, Hispanic, all other)
+found in ACS PUMS recent immigrants.
+
+Then, we distribute emigration within each race/ethnicity and move type by further demographics,
+according to the distributions of these demographics in a resample of the corresponding ACS immigrant population,
+with perturbation as described in the :ref:`perturbation section below <census_prl_perturbation>`.
+Note that in the case of household moves, these are the demographics **of the immigrant's household's reference person**,
+while for the other two types they are demographics of the immigrant themselves.
+
+Finally, we calculate the rates of people emigrating per year of person-time "at risk":
+
+* The "at risk" population for household moves is people living in non-GQ households.
+* The "at risk" population for GQ person moves is people living in GQ.
+* The "at risk" population for non-reference-person moves is people living in non-GQ households who
+  are not the reference person in their household.
+
+In order to mitigate the sampling noise in ACS PUMS stratified by all of these demographic characteristics,
+we calculate the denominator for the rate from a resample of the "at risk" population, with perturbation.
+
+Simulation strategy
+^^^^^^^^^^^^^^^^^^^
+
+Emigration events are modeled as happening to an at-risk population at a certain rate.
+They are constant across time in the simulation.
+
+Households and individuals selected to have emigration events should remain in the simulation, but their
+state and PUMA attributes should be set to a placeholder value that signifies they are
```

Comment:
by "state" do you mean US state, or simulation-state? 

could you specify which columns become placeholders vs. which ones continue to be "live"?

Reply:
US state; will revise to clarify.

My intention here is that columns besides the ones explicitly mentioned continue to have non-placeholder values. Then, separately, in many of the other components we will need to specify that they only act on simulants in the US. I have done that in this PR. This is a bit tricky and error-prone, but a) I *think* you will need to implement it this way, if it works anything like how death works in Vivarium, and b) we would want certain events to continue to happen, e.g. death, if we ever add the ability to bring simulants back from abroad.

I will spell this out more.